extensions/sql-database-projects/src/models/project.ts

```diff
@@ -70,6 +70,24 @@ export class Project {
 					this.files.push(this.createProjectEntry(folderElements[f].getAttribute(constants.Include), EntryType.Folder));
 				}
 			}
+
+			// find all pre-deployment scripts to include
+			const preDeploy = itemGroup.getElementsByTagName(constants.PreDeploy);
+			for (let p = 0; p < preDeploy.length; p++) {
+				this.files.push(this.createProjectEntry(preDeploy[p].getAttribute(constants.Include), EntryType.File));
```

Comment:
since only one is realy allowed - should we pop up a toast notif if more than one are found with pre/post tag?

Reply:
Maybe throw error, since it's an invalid state since we wouldn't know which to use.

Reply:
Added an informational message here, if pre/post script count is more than one.

Reply:
Updated it to a warning message.